payment/order.go

```diff
@@ -10,6 +12,26 @@ import (
 	macaroon "gopkg.in/macaroon.v2"
 )
 
+// NullString is a type that lets ya get a null field from the database
+type NullString struct {
+	sql.NullString
+}
+
+// MarshalJSON for NullString
+func (ns *NullString) MarshalJSON() ([]byte, error) {
+	if !ns.Valid {
+		return []byte("null"), nil
+	}
+	return json.Marshal(ns.String)
+}
+
+// UnmarshalJSON unmarshalls NullString
+func (ns *NullString) UnmarshalJSON(data []byte) error {
+	ns.String = strings.Trim(string(data), `"`)
+	ns.Valid = true
```

Comment:
What if you get `"location": null`... I believe that will insert a "valid" empty string into the database eventually.  Suggest we check for null values:

```go
if len(data) == 0 {
    ns.Valid = false
}
````

Reply:
https://github.com/brave-intl/bat-go/pull/308/commits/7152ca89e9402d3aa50204ded7d1bd68d331e75f